portfolio/src/main/webapp/index.html

```diff
@@ -7,15 +7,20 @@
     <script src="script.js"></script>
   </head>
   <body>
+		<div id="header" class="center">
```

Comment:
So I guess the problem here is you are using tabs that aren't getting converted to spaces. The rest of the doc is using spaces instead of tabs. So this new section is unaligned here, but it might not have been on your computer because tabs are different lengths depending on your settings. I have my editor setup to automatically convert tabs to spaces.

Reply:
I forgot to do that, thanks! I converted the tabs into spaces and committed again.